src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -57,4 +64,22 @@ public static UbiquitousLanguageAsFile forFormat(String outputFormat, String out
     }
     throw new IllegalArgumentException("Supported file formats are 'json', 'yml'.");
   }
+
+  public class Manifest {
```

Comment:
private static final class

Reply:
If "Manifest" becomes static, we don't have access to the attribute of "UbiquitousLanguageAsFile" like "extension". And we need it currently.